apps/users/models.py

```diff
@@ -464,12 +464,12 @@ def special_collection(self, type_, defaults):
         return c
 
     @contextmanager
-    def activate_lang(self):
+    def activate_lang(self, override_lang):
```

Comment:
You need to revert the changes in this file, which aren't useful anymore I believe